game/rooms/room15/esc/room15_unused.esc

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Why is this called room15_unused?

Reply:
The intention was to show that the file was not used... but having no file is better :)